src/main/java/com/commercetools/sync/categories/utils/CategoryUpdateActionUtils.java

```diff
@@ -251,7 +251,8 @@
             return AssetsUpdateActionUtils.buildAssetsUpdateActions(
                 oldCategory.getAssets(),
                 newCategory.getAssets(),
-                new CategoryAssetActionFactory(syncOptions));
+                new CategoryAssetActionFactory(syncOptions),
```

Comment:
could be formatted better for readability.

Reply:
954b98ce